### DISCUSSION TITLE
Install Harbor on server with multiple IP and one IP's 443 port already used.

### DISCUSSION THREAD

**Discussion by zhihuadong** [goharbor/harbor#18561] [Q&A]
I am trying to install harbor on a server with another service enabled on another IP's 443 port. 
./install.sh  failed to start ngnix due to 0.0.0.0:443 already used.
I changed the docker-compose.yml file   modified  the proxy container's ports adding harbor  ip address in the port mapping
  ports:
      - xx.xx.xx.xx:80:8080
      - xx.xx.xx.xx:443:8443
Seems all container can start, no error found in logfiles. But browse to harbor  site only show a circling wheel , maybe waiting something. How do I debug on what's wrong.  Or is this supported ?

**Comment by stonezdj** [comment]
Update the docker-compose file doesn't work, you should update the harbor.yaml file and rerun the ./prepare or ./install.sh command. and restart Harbor.
